src/audiomixerboard.cpp

```diff
@@ -1335,12 +1335,16 @@ void CAudioMixerBoard::ApplyNewConClientList ( CVector<CChannelInfo>& vecChanInf
 
 void CAudioMixerBoard::SetFaderLevel ( const int iChannelIdx, const int iValue )
 {
+    // Proposed change: if iChannelIdx is -1 and our own channel ID is a valid index
+    // then we adjust our own fader level:
+    const int iTheChannelIdx = ( iChannelIdx == INVALID_INDEX ) ? iMyChannelID : iChannelIdx;
+
     // only apply new fader level if channel index is valid and the fader is visible
-    if ( ( iChannelIdx >= 0 ) && ( iChannelIdx < MAX_NUM_CHANNELS ) )
+    if ( ( iTheChannelIdx >= 0 ) && ( iTheChannelIdx < MAX_NUM_CHANNELS ) )
```

Comment:
TheChannelIdx is also not expressive.

Reply:
It's the channel index that will be used.  `iChannelIdx` is already taken.  I could use `i`?  Any other suggestions?

Reply:
iChannelIdxToModify?

Reply:
`iRealChannelIdx` chosen.